source/CardGames.py

```diff
@@ -280,34 +280,125 @@ def winnnerCheck(self, house, player):
+    [   [480,500,120,35,20],quit_text ],#   ~~~
+    [   [480,350,120,35,20],decks_text ],
+    [   [480,100,120,35,20],play_text ],
+    [   [480,200,120,35,10],early_text ]
+]
+
+
+in_menu=True 
+while in_menu: 
+      
+    for ev in pygame.event.get(): 
+          
+        if ev.type == pygame.QUIT: 
+            pygame.quit() 
+              
+        #checks if a mouse is clicked 
+        if ev.type == pygame.MOUSEBUTTONDOWN: 
+              
+            element = buttonlist[0]#first button   ~~~
+            if element[0][0] <= mouse[0] <= element[0][0]+element[0][2] and element[0][1] <= mouse[1] <= element[0][1]+element[0][3]:#   ~~~ 
+                pygame.quit()#   ~~~
+
+            element = buttonlist[1]#second button
+            if element[0][0] <= mouse[0] <= element[0][0]+element[0][2] and element[0][1] <= mouse[1] <= element[0][1]+element[0][3]: 
+                if deck_num==1:
+                    deck_num=2
+                elif deck_num==2:
+                    deck_num=4
+                elif deck_num==4:
+                    deck_num=8
+                elif deck_num==8:
+                    deck_num=1
+
+            element = buttonlist[2]#third button
+            if element[0][0] <= mouse[0] <= element[0][0]+element[0][2] and element[0][1] <= mouse[1] <= element[0][1]+element[0][3]:
+                in_menu=False
+            
+            element = buttonlist[3]#fouth button
+            if element[0][0] <= mouse[0] <= element[0][0]+element[0][2] and element[0][1] <= mouse[1] <= element[0][1]+element[0][3]:
+                if not early_shuffle:
+                    early_shuffle = True
+                else:
+                  early_shuffle = False
+
+    # background color 
+    screen.fill((60,25,60)) 
+      
+    # stores the (x,y) coordinates into 
+    # the variable as a tuple 
+    mouse = pygame.mouse.get_pos() 
+      
+    # if mouse is hovered on a button it changes to a lighter shade 
+    for element in buttonlist:
+        if element[0][0] <= mouse[0] <= element[0][0]+element[0][2] and element[0][1] <= mouse[1] <= element[0][1]+element[0][3]: 
+            pygame.draw.rect(screen,color_light,[element[0][0],element[0][1],element[0][2],element[0][3]]) 
+          
+        else: 
+            pygame.draw.rect(screen,color_dark,[element[0][0],element[0][1],element[0][2],element[0][3]]) 
+      
+        # superimposing the prompt text onto our button 
+        screen.blit(element[1], (element[0][0]+element[0][4],element[0][1])) 
+
+    #button values displayed
+
+    #deck value
+    deckCount = smallfont.render( str(deck_num), True , color)
+    element = buttonlist[1]#second button
+    screen.blit(deckCount, (element[0][0]+element[0][4]+30,element[0][1]+40))
+    #shuffle style
+    if early_shuffle:
+        indeck_limit=52*deck_num//2
+    else:
+        indeck_limit=0
+    earlyValue = smallfont.render( "at: "+str(indeck_limit), True , color)
```

Comment:
Maybe provide more info regarding what shuffle means. For example, 
earlyValue = smallfont.render( "Shuffle at "+str(indeck_limit)+" cards left", True , color)
Of course then,
screen.blit(earlyValue, (element[0][0]+element[0][4]-80,element[0][1]+40)) 
would have to be updated like so to be centered.

And/or in the future add the ability to hover over the buttons so a text box is displayed that tells the user how changing the specific setting will impact gameplay.

Reply:
good idea. I have updated the code accordingly